### PR TITLE
Cleaning up makefile.shade

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @victorhurdugaci

File: _use-release-management.shade [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        {
+          "Coherence",
+          "Coherence-Signed",
+          "dnvm",
+          "Entropy",
+          "Setup",
+          "libuv-build",
+        };
+
+        Exec("cmd", "/C dnu restore", @"tools\PinVersion");
```
Comment (victorhurdugaci) [inline _use-release-management.shade]: dnu?

**Comment by victorhurdugaci** [comment]
Assuming you just moved code :shipit: but if the point of this PR is to also make the code work, then let me know and I'll add comments for that

File: _use-release-management.shade (hunk 1 above)
Comment (pranavkm) [inline _use-release-management.shade]: Yeah I just copied the code around so `makefile.shade` is easier to navigate. We're going to a dry run of release next week and I'll clean up the tools at that point.

**Pull request closed without merging**